content/en/v3/admin/guides/spot-instances/_index.md

```diff
@@ -0,0 +1,134 @@
+---
+title: Spot Instances For Builds
+linktitle: Spot Instances For Builds
+type: docs
+description: How to use spot instances to handle all Tekton builds
+weight: 120
+aliases:
+  - /v3/guides/spot-instances-for-builds
+---
+
+Using spot instances for builds is a great way to detach your builds (reducing workload on your system pods) and save money.
+
+Jenkins-x supports a [Tekton](https://Tekton.dev/) helm chart that lets configuring spot instances for builds really easy. 
```

Comment:
Jenkins X, Initially I thought it is JenkinsX or Jenkins-X, but turns out it is Jenkins X :grin: